examples/app-secure-aggregation/README.md

```diff
@@ -1,3 +1,10 @@
+---
+title: Example Flower App with Secure Aggregation
+labels: [basic, vision, fds]
+dataset: [CIFAR-10 | https://huggingface.co/datasets/uoft-cs/cifar10]
```

Comment:
This example doesn't use a dataset (just `NumPy` array simulating a model being sent). @panh99 what should we do with this example?

Reply:
I made it so it is possible to leave the dataset empty